report/assets/styles.css

```diff
@@ -561,6 +562,36 @@
   border-bottom: var(--score-icon-size) solid var(--color-fail);
 }
 
+.lh-audit--error .lh-audit__score-icon,
+.lh-metric--error .lh-metric__icon {
+  background-image: var(--error-icon-url);
+  background-repeat: no-repeat;
+  width: calc(var(--score-icon-size) * 2.4);
+  height: calc(var(--score-icon-size) * 2.4);
+  background-position: center;
+  margin-right: 0;
+  border: none;
+}
+
+.lh-gauge__wrapper--fail .lh-gauge--error {
+  background-image: var(--error-icon-url);
+  background-repeat: no-repeat;
+  background-position: center;
+  top: var(--score-container-padding);
+}
+
+.lh-sticky-header .lh-gauge__wrapper--fail .lh-gauge--error {
+  height: calc(var(--score-icon-size) * 3.25);
+  background-position: center;
+  top: calc(var(--score-container-padding) / 2);
+}
+
+.lh-gauge__wrapper--fail:not([href]) .lh-gauge--error {
+  background-image: var(--error-icon-url);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
This rule doesn't actually change anything now and can safely be deleted

Reply:
sounds good, will remove it.